index.html

```diff
@@ -331,44 +376,48 @@ <h2 class="heading heading-sm main-footer__heading-sm">
               <span>Social</span>
             </h2>
             <div class="main-footer__social-cont">
-              <a target="_blank" rel="noreferrer" href="https://www.linkedin.com/in/mastoora-turkmen/">
-                <img
-                  class="main-footer__icon"
-                  src="./assets/img/linkedin-ico.png"
-                  alt="icon"
-                />
+              <a
+                class="main-footer__icon"
+                target="_blank"
+                title="linkedin"
+                rel="noopener noreferrer"
+                href="https://www.linkedin.com/in/mastoora-turkmen/"
+              >
+                <i class="fa-brands fa-linkedin fa-2xl"></i>
```

Comment:
this how we use font awesome icon <i class="fa-brands fa-linkedin fa-2xl"></i>
[https://fontawesome.com/docs/web/setup/get-started](https://fontawesome.com/docs/web/setup/get-started) here you can check it yourself